File: apps/problems/Find-Prime-Numbers/boilerplate-full/function.rs
use std::fs::read_to_string;
    use std::io::{self};
    use std::str::Lines;
    
    ##USER_CODE_HERE##
    
    fn main() -> io::Result<()> {
      let input = read_to_string("/dev/problems/Find-Prime-Numbers/tests/inputs/##INPUT_FILE_INDEX##.txt")?;
      let mut lines = input.lines();
      let limit: i32 = lines.next().unwrap().parse().unwrap();
      let result = findPrimes(limit);
      println!("{}", result);
      Ok(())
    }
    

File: apps/problems/Find-Prime-Numbers/boilerplate-full/function.cpp
#include <iostream>
    #include <fstream>
    #include <vector>
    #include <string>
    #include <sstream>
    #include <climits>
    
    ##USER_CODE_HERE##
    
    int main() {
      std::ifstream file("/dev/problems/Find-Prime-Numbers/tests/inputs/##INPUT_FILE_INDEX##.txt");
      std::vector<std::string> lines;
      std::string line;
      while (std::getline(file, line)) lines.push_back(line);
    
      file.close();
      int limit;
  std::istringstream(lines[0]) >> limit;
      std::list<int> result = findPrimes(limit);
      std::cout << result << std::endl;
      return 0;
    }
    